Makefile

```diff
@@ -351,9 +351,16 @@ multitenancytests:
 	-v $(PWD):/go/src/$(PACKAGE_PATH_AKO) $(GO_IMG_TEST) \
 	$(GOTEST) -mod=vendor $(PACKAGE_PATH_AKO)/tests/multitenancytests/... -failfast -timeout 0 -coverprofile cover-21.out -coverpkg=./...
 
+.PHONY: namingtests
```

Comment:
can we change the name to encoding tests or url tests.

Reply:
@aaha97 I have changed it to urltests.